### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by coder11**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,76 @@
...
+
+### dev
+
+**Unstable** Several nodes running on the same physical host. Used a lot to test things out. 
+
+### stage
+
+Environment primarily used for staging deployments of various Fluence services.
+
+### FaaS Net
```
Comment (folex) [inline README.md]: I think it is better to call it `Test Net`

File: README.md [hunk 2]
```diff
@@ -1 +1,76 @@
...
+| Security & Accounting |                          Blockchain                          |          Whole network           |        Licenses & payments        |                          substrate?                          |
+
+<img alt="aquamarine scheme" align="center" src="doc/stack.png"/>
+
+
+
+
+## Contributing
+
+While the project is a still in the early stage of development, you are welcome to track progress and contribute. At the current moment we don't have detailed instructions on how to join development or which code guidelines to follow. However, you can expect more info to appear soon enough. In the meanwhile, check out the [basic contributing rules](https://github.com/fluencelabs/fluence/blob/trustless_computing/CONTRIBUTING.md).
```
Comment (folex) [inline README.md]: The link to contributing is strange, it points to the `trustless_computing` tag. Maybe point to master instead?
https://github.com/fluencelabs/fluence/blob/master/CONTRIBUTING.md

**Review by folex — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (coder11) [reply README.md]: Hhhm. It might be better to just remove contributing file since this repository is not really meant for contributions

Comment (coder11) [reply README.md]: Either way, I think we should the same contribution guidelines for all of the Fluence Project repositories... May be it's better to extract this info into readme.io?

Comment (folex) [reply README.md]: Not sure, I think the contribution flow needs more than just CONTRIBUTING.md anyway. 

I think removing is a good choice right now.

Comment (coder11) [reply README.md]: Btw, I've copy-pasted it from fluence rep

File: README.md (hunk 1 above)
Comment (coder11) [reply README.md]: What's the difference between test and stage then?

**Pull request merged**